services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowCrudV2Spec.groovy

```diff
@@ -826,6 +829,25 @@ class FlowCrudV2Spec extends HealthCheckSpecification {
         [vlanFlow, defaultFlow].each { flow -> flowHelper.deleteFlow(flow.flowId) }
     }
 
+    @Ignore("number of ticket will be added soon")
+    @Tags(VIRTUAL)
+    def "System doesn't ignore encapsulationType when flow is created with ignoreBandwidth = true"() {
+        given: "Two active switches"
+        def swPair = topologyHelper.getNeighboringSwitchPair()
```

Comment:
```
topologyHelper.allSwitchPairs.find { [it.src, it.dst].any { !northbound.getSwitchProperties(it.dpId).supportedEncapsulation.contains(FlowEncapsulationType.VXLAN.toString().toLowerCase()) } }
```
Or even better: explicitly make src/dst switch to not support vxlan encap. Remove Virtual tag.

Reply:
fixed